tests/test_parallels.py

```diff
@@ -257,9 +268,15 @@ def test_parallel_synchronisation():
     assert(success.status == py_trees.common.Status.SUCCESS)
     print("success_every_second.status == py_trees.common.Status.SUCCESS")
     assert(success_every_second.status == py_trees.common.Status.SUCCESS)
-    # check that succeess did not get ticked
-
-    py_trees.tests.tick_tree(root, 3, 3, visitor, print_snapshot=True)
+    print("success [id: {}] did not get ticked [snapshot: {}]".format(success.id, [str(ident) for ident in snapshot_visitor.nodes.keys()]))
+    assert(success.id not in snapshot_visitor.nodes)
+
+    snapshot_visitor.initialise()
+    py_trees.tests.tick_tree(
+        root, 3, 3,
+        visitors=[debug_visitor, snapshot_visitor],
+        print_snapshot=True
+    )
 
     print("\n--------- Assertions ---------\n")
     print("root.status == py_trees.common.Status.RUNNING")
```

Comment:
I think we should ensure this after the third tick:
`assert success.id in snapshot_visitor.nodes.keys()`